### PR TITLE
feat(groovyw): add command to create dependency dot file

### PR DESCRIPTION
**Pull request by jdrueckert**
- integrates all direct and recursive dependencies of the given module (or all if "all" or "*" or "")
- can be called, e.g. via `./groovyw module createDependencyDotFile <moduleName>`
- can be visualized e.g. using graphviz using `dot -Tsvg dependencies.dot > dependencies.svg`
- cleans up formatting (kudos to the IDE)

### CODE REVIEW

**Comment by jdrueckert** [comment]
Visualization example via graphviz on JoshariasSurvival:

image

**Comment by pollend** [comment]
really nice way to visualize module dependencies: 

image

File: config/groovy/common.groovy [hunk 1]
```diff
@@ -466,32 +499,32 @@ class common {
...
+    static final String CYAN = "\u001B[36m"
+    static final String WHITE = "\u001B[37m"
+
+    static final String DARK_GRAY = "\u001B[1;30m"
+    static final String LIGHT_RED = "\u001B[1;31m"
+    static final String LIGHT_GREEN = "\u001B[1;32m"
+    static final String LIGHT_YELLOW = "\u001B[1;33m"
+    static final String LIGHT_BLUE = "\u001B[1;34m"
+    static final String LIGHT_PURPLE = "\u001B[1;35m"
+    static final String LIGHT_CYAN = "\u001B[1;36m"
```
Comment (pollend) [inline config/groovy/common.groovy:527]: do we care about loosing this formatting with the text?

**Comment by pollend** [comment]
also could you create a short wiki entry with how to use this.

File: config/groovy/common.groovy (hunk 1 above)
Comment (jdrueckert) [reply config/groovy/common.groovy:527]: Didn't check all of the formatting changes the IDE did tbh, but true, that should be retained.

**Comment by jdrueckert** [comment]
> also could you create a short wiki entry with how to use this.

yup will do

File: config/groovy/common.groovy (hunk 1 above)
Comment (jdrueckert) [reply config/groovy/common.groovy]: done

**Comment by jdrueckert** [comment]
> also could you create a short wiki entry with how to use this.

done: https://github.com/MovingBlocks/Terasology/wiki/Module-Dependencies#visualizing-module-dependencies

**Pull request merged**